common/src/main/scala/akka/contrib/persistence/mongodb/MongoPersistence.scala

```diff
@@ -45,6 +51,14 @@ trait CanDeserializeJournal[D] {
   def deserializeDocument(document: D)(implicit serialization: Serialization, system: ActorSystem): Event
 }
 
+trait CanSuffixCollectionNames {
+  def getSuffixFromPersistenceId(persistenceId: String): String
+}
+
```

Comment:
Maybe it's better to remove this, and default to something that is unset - I see that's handled below in `getSuffixFromPersistenceId`